addons/web_editor/static/src/js/backend/html_field.xml

```diff
@@ -10,7 +10,7 @@
                 <div  t-ref="readonlyElement" class="o_readonly" t-out="markupValue" />
             </t>
         </t>
-        <t t-else="">
+        <div t-else="" t-ref="spellcheck">
```

Comment:
I wouldn't add a div here (you can use t-ref twice as long as they are mutually exclusive)

Reply:
Because the editable element can be a textarea, or a contenteditable div initiated by a HtmlFieldWysiwygAdapterComponent (etc.), I added this div as a common parent.

Should I search another way, maybe wrap the adapter in a div, using the same ref?